openpdf/pom.xml

```diff
@@ -58,8 +58,39 @@
             <resource>
                 <directory>src/main/resources-filtered</directory>
                 <filtering>true</filtering>
+                <includes>
+                    <include>**/version.txt</include>
```

Comment:
If I understand your intention correctly, current OpenPDF version will be saved at specified files during build. If so, these files must be added to .gitignore. Or they will be generated not during compilation phase?